appservice/src/createAppService/SiteNameStep.ts

```diff
@@ -27,13 +27,26 @@ export class SiteNameStep extends AzureNameStep<IAppServiceWizardContext> {
         if (wizardContext.environment.name === 'Azure') {
             // Unfortunately, the environment object doesn't have the url we need for this placeholder. Might be fixed in the new sdk: https://github.com/microsoft/vscode-azuretools/issues/510
             // For now, we'll only display this placeholder for the most common case
-            const namePlaceholder: string = wizardContext.newSiteKind === AppKind.functionapp ? localize('funcAppName', 'function app name') : localize('webAppName', 'web app name');
+            let namePlaceholder: string;
+            if (wizardContext.newSiteKind === AppKind.functionapp) {
+                namePlaceholder = localize('funcAppName', 'function app name');
+            } else if (wizardContext.newSiteKind === AppKind.workflowapp) {
```

Comment:
It is better to do a includes/contains check rather a direct equals since siteKind can contain more values like - 'kubernetes', 'container'. Same comment in line:44